build.gradle

```diff
@@ -156,6 +156,19 @@ dependencies {
         exclude module: 'kryo' // use Spark's version
     }
 
```

Comment:
Do you need this as well (from protected)?

    // Comment the next line to disable native code proxies in Spark MLLib
    compile('com.github.fommil.netlib:all:1.1.2')